packages/sheet/src/sheet.ts

```diff
@@ -74,11 +74,11 @@ export class Sheet {
     css: string,
     breakpoint: string
   ): void {
-    css = css.replace(cssPropertyRegex, "");
+    css = css.replace(cssPropertyRegex, "$1$2");
     const mediaCss =
       `@media (min-width: ${breakpoint}) { .${className} { ${css} } }`.replace(
         removeSpacesExceptInPropertiesRegex,
-        ""
+        "$1$2$3"
       );
```

Comment:
@myzkyy Is it possible to extract this whitespace removal logic as a function?

example:

```ts
const removeSpacesExceptInProperties = (str: string) => str.replace(removeSpacesExceptInPropertiesRegex, "$1$2$3")
```

Reply:
@kotarella1110 Done.